### DISCUSSION TITLE
Auch Erzeuger möglich?

### DISCUSSION THREAD

**Discussion by wolliboy** [camueller/SmartApplianceEnabler#192] [Allgemein]
Moin.
Der SAE ist ja dafür da um Verbraucher in das Sunny Portal zu bringen.
Jetzt frage ich mich ob auch möglich wäre einen Erzeuger über den SAE in das Sunny Portal zu bringen?
Der Unterschied zum Verbraucher ist ja nur das Minuszeichen bei der Leistung.
So könnte man ein kleines Windrad oder Balkonanlage mit einbinden, was jetzt keine Wechselrichter von SMA sind.

Probiert habe ich es schon, aber die Leistung meiner "Balkonanlage" wird positiv als Verbraucher dargestellt und nicht negativ als Erzeuger. Kann natürlich auch einfach daran liegen, dass meine Messung es nicht hergibt. 
Ich habe probiert:
Leistungsmessung mit Fritz DECT 200 und 
Sonoff POWR2. 
Und dann gibt es im SAE ja noch das Feld für den Umrechnungsfaktor, aber das lässt keine negativen Werte zu. 

Grüße

**Comment by camueller** [comment]
Nein, das SEMP-Protokoll, über das SAE und SHM miteinander kommunizieren unterstützt keine Einbindung von Erzeugern.
